packages/metadataform-extension/package.json

```
@@ -0,0 +1,61 @@
+{
+  "name": "@jupyterlab/metadataform-extension",
+  "version": "4.0.0-alpha.14",
+  "description": "A helper to build form for metadata",
+  "keywords": [
+    "jupyter",
+    "jupyterlab",
+    "jupyterlab-extension"
+  ],
+  "homepage": "https://github.com/jupyterlab/jupyterlab",
+  "bugs": {
+    "url": "https://github.com/jupyterlab/jupyterlab/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jupyterlab/jupyterlab.git"
+  },
+  "license": "BSD-3-Clause",
+  "author": "Project Jupyter",
+  "sideEffects": [
+    "style/**/*"
+  ],
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "style": "style/index.css",
+  "directories": {
+    "lib": "lib/"
+  },
+  "files": [
+    "lib/**/*.{d.ts,eot,gif,html,jpg,js,js.map,json,png,svg,woff2,ttf}",
+    "schema/*.json",
+    "style/**/*.{css,eot,gif,html,jpg,json,png,svg,woff2,ttf}",
+    "style/index.js"
+  ],
+  "scripts": {
+    "build": "tsc",
+    "clean": "rimraf lib && rimraf tsconfig.tsbuildinfo",
+    "watch": "tsc -w --listEmittedFiles"
+  },
+  "dependencies": {
+    "@jupyterlab/application": "^4.0.0-alpha.14",
+    "@jupyterlab/metadataform": "^4.0.0-alpha.14",
+    "@jupyterlab/notebook": "^4.0.0-alpha.14",
+    "@jupyterlab/settingregistry": "^4.0.0-alpha.14",
+    "@jupyterlab/translation": "^4.0.0-alpha.14",
+    "@jupyterlab/ui-components": "^4.0.0-alpha.29",
```

These can be bumped as well:

```suggestion
    "@jupyterlab/application": "^4.0.0-alpha.15",
    "@jupyterlab/metadataform": "^4.0.0-alpha.15",
    "@jupyterlab/notebook": "^4.0.0-alpha.15",
    "@jupyterlab/settingregistry": "^4.0.0-alpha.15",
    "@jupyterlab/translation": "^4.0.0-alpha.15",
    "@jupyterlab/ui-components": "^4.0.0-alpha.30",
```